## Handling errors with Asynchronous code
### Practice project

*Making a function cookBeanSouffle() which returns a promise that resolves or rejects randomly.When it resolves, the promise resolves with a value of 'Bean Souffle' and, when it rejects,it rejects with a value of 'Dinner is ruined!'*
